Live_data_update.sql

```diff
@@ -0,0 +1,10 @@
+https://trello.com/c/CKuDXAbq
+update cqc."Establishment" set "MainServiceFKValue"=20 where "EstablishmentID"=340 and "LocationID"='1-5254278626';
+
+
+
+
+https://trello.com/c/oeYhp3Ui
+
+update cqc."Establishment" set "IsRegulated"=True where "EstablishmentID"=337;
+
```

Comment:
Unfortunately with record 337 LocationID blank. So its not possible update the location ID.

Thanks
Shakir